### PR TITLE
WIP fix for upgrading from the old site SW

### PR DESCRIPTION
**Pull request by developit**
Upgrading from the old site for those who've previously visited it currently requires a refresh, since the `sw.js` did not use `skipWaiting()`.

### CODE REVIEW

**Comment by prateekbh** [comment]
Need to think how will the modern browser users upgrade from `sw.js` to `sw-esm.js`

**Comment by prateekbh** [comment]
Turns out its takes care of by the browser 👍

File: src/sw-fix.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+// this gets appended to the built sw
+
+/*global clients*/
+
+// eslint-disable-next-line
+var storage = (function(e) {
+	'use strict';
```
Comment (prateekbh) [inline src/sw-fix.js]: Travis is unhappy with this

Comment (developit) [reply src/sw-fix.js]: hmm - this was previously all on one line, which meant the ESLint ignore on L5 applied to the whole thing.

**Comment by developit** [comment]
(fixed elsewhere!)

**Pull request closed without merging**